src/components/SignUpForm/index.js

```diff
@@ -74,6 +74,8 @@ const SignUpFormUI = (props) => {
 
   const showInputPhoneNumber = validationFields?.fields?.checkout?.cellphone?.enabled ?? false
 
+  const isBusinessSignUp = (configs?.business_signup_allow?.value === "1" ? true : false)
```

Comment:
use this validation: `const isBusinessSignUp = configs?.business_signup_allow?.value === "1"`